dev-infrastructure/scripts/istio.sh

```diff
@@ -1,6 +1,13 @@
 #!/bin/bash
 
 set -euo pipefail
+ISTIO_NAMESPACE="aks-istio-system"
```

Comment:
Can we use `az aks mesh versions`?

Reply:
use that for what?

Reply:
For checking what versions of the `istio` control plane are available, without having to assume things like where the AKS plugin happens to run them, how they are annotated, whatever. Not sure we care about any of those implementation details, and we certainly do not care about making sure that we know how many replicas the plugin decides to run. What if the plugin thinks 5 replicas are necessary? Or 3? Or just 1?

Reply:
the az aks mesh ```az aks mesh get-revisions --location``` only check the avilable versions to upgrade to in the certain location. it does not indicate the running istio in the current cluster